src/client/components/pages/RoomAdmin/RoomAdmin.tsx

```diff
@@ -258,31 +251,34 @@ const RoomAdmin: FunctionComponent = (): ReactElement => {
                   <TableRow isStriped>
                     <TableHeadingCell scope="col">
                       <Dropdown
-                        options={
-                          [{ value: 'All', label: 'All' }, ...campusDropDown]
-                        }
-                        name="filterByCampusName"
-                        id="filterByCampusName"
-                        label="The table will be filtered by selected campus name"
-                        isLabelVisible={false}
                         hideError
+                        id="campus-filter"
+                        label="Change to filter the list of meetings by campus"
+                        isLabelVisible={false}
                         onChange={
-                          (event:React.ChangeEvent<HTMLInputElement>) => {
-                            setCampusFilter(event.currentTarget.value);
+                          (evt: ChangeEvent<HTMLSelectElement>): void => {
+                            setCampusFilter(evt.target.value as CAMPUS);
                           }
                         }
+                        name="campus-filter"
+                        value={campusFilter}
+                        options={
+                          Object.values(CAMPUS)
+                            .map((value) => ({ value, label: value }))
+                        }
                       />
+
                     </TableHeadingCell>
                     <TableHeadingCell scope="col">
                       <Dropdown
+                        hideError
+                        id="building-filter"
+                        label="Change to filter the list of meetings by building"
```

Comment:
The room admin page is filtering rooms, not meetings, so this text should be updated.